playbot/java/src/main/java/com/RunBot.java

```diff
@@ -0,0 +1,12 @@
+package com;
+
+public class RunBot {
+
+    public static void main(String[] args){
+        String host = "localhost";
+        int port = 21000;
```

Comment:
is there a way to get host and port from args?
sorry but i don't really read in this lang ;)

Reply:
Sure there is :)
Just check the new version.